src/components/Shared/Filters/NullState/NullState.tsx

```diff
@@ -68,6 +69,8 @@ interface Props {
   page: 'contact' | 'task';
   totalCount: number;
   filtered: boolean;
+  title?: string;
```

Comment:
The buttons should be removed when there
<img width="1352" alt="Screenshot 2024-08-21 at 1 02 09 PM" src="https://github.com/user-attachments/assets/cfc90023-dfad-4174-8980-942e1e88d7e2">
 are no `Given` contacts.

Reply:
Fix on future PR